controllers/cloud.redhat.com/providers/iqe/impl.go

```diff
@@ -119,7 +83,49 @@ func CreateIqeJobResource(cache *rc.ObjectCache, cji *crd.ClowdJobInvocation, en
 		TerminationMessagePolicy: core.TerminationMessageReadFile,
 	}
 
+	return c
+}
+
+func createSeleniumContainer(name string, nn types.NamespacedName, cji *crd.ClowdJobInvocation, env *crd.ClowdEnvironment, app *crd.ClowdApp) core.Container {
+	// create env vars
+	envVars := []core.EnvVar{
+		{Name: "HOME", Value: "/home/selenium"},
+	}
+
+	// set image tag
+	image := env.Spec.Providers.Testing.Iqe.UI.Selenium.ImageBase
+	if image == "" {
+		image = "quay.io/redhatqe/selenium-standalone"
+	}
+	tag := env.Spec.Providers.Testing.Iqe.UI.Selenium.DefaultImageTag
+	if tag == "" {
+		tag = "ff_91.5.1esr_gecko_v0.30.0_chrome_98.0.4758.80"
```

Comment:
I have a little concern about the hardcoding tag name here.

Reply:
It's only a default, but perhaps it would be better to make the default part of clowders config? It's not the only place we hardcode images in Clowder thuogh.

Reply:
We will deal w/ default values in a follow-up PR. As Pete mentioned we have a few cases of these hard-coded values and we'll move them all to config via a different PR. The default image tag for selenium is actually normally going to be controlled by the value set in the ClowdEnvironment. This hard-coded value here is just a fallback to ensure it is never empty by accident.